rswitcher@nixdev.com/files/rswitcher@nixdev.com/rswitcher

```diff
@@ -24,15 +24,19 @@ check_args()
 
 create_resolution()
 {
+# Using Luxodius python script for scaling 
+# https://bbs.archlinux.org/viewtopic.php?id=278808
+  
+    .local/share/cinnamon/applets/rswitcher@nixdev.com/scale.py $SCALINGFACTOR
```

Comment:
Have you tested if this will work? It should need to start with `~/` to execute successfully.

Reply:
It works but starting with ~/ makes more sense. Thanks for the review! :+1: